monai/transforms/utility/dictionary.py

```diff
@@ -1740,6 +1741,69 @@ def __call__(self, data: Mapping[Hashable, NdarrayOrTensor]) -> dict[Hashable, N
         return d
 
 
+class ApplyTransformToPointsd(MapTransform, InvertibleTransform):
+    """
+    Dictionary-based wrapper of :py:class:`monai.transforms.ApplyTransformToPoints`.
+    The input coordinates are assumed to be in the shape (C, N, 2 or 3),
+    where C represents the number of channels and N denotes the number of points.
+
+    Args:
+        keys: keys of the corresponding items to be transformed.
+            See also: monai.transforms.MapTransform
+        refer_key: The key of the reference item used for transformation.
+            It can directly refer to an affine or an image from which the affine can be derived.
+        dtype: The desired data type for the output.
+        affine: A 3x3 or 4x4 affine transformation matrix applied to points. This matrix typically originates
+            from the image. For 2D points, a 3x3 matrix can be provided, avoiding the need to add an unnecessary
+            Z dimension. While a 4x4 matrix is required for 3D transformations, it's important to note that when
+            applying a 4x4 matrix to 2D points, the additional dimensions are handled accordingly.
+            The matrix is always converted to float64 for computation, which can be computationally
+            expensive when applied to a large number of points.
+        invert_affine: Whether to invert the affine transformation matrix applied to the points. Defaults to ``True``.
+            Typically, the affine matrix is derived from the image, while the points are in world coordinates.
+            If you want to align the points with the image, set this to ``True``. Otherwise, set it to ``False``.
+        affine_lps_to_ras: Defaults to ``False``. Set this to ``True`` if all of the following are true:
```

Comment:
Can you please make a note here that we need to check this point a bit more. It makes sense that `affine_lps_to_ras` is `True` when **all** the statements are true. But MONAI also used readers other than ITK. So what happens when we use nibabel as the image reader. 
Or maybe add few more guidelines as in what happens when only a few conditions are met. 

@ericspod Just to confirm that the world coordinate system is RAS but nifti has an LPS coordinate system right?

Reply:
It will only be True when all the statements are true so all other case will be False here.

Reply:
We're assuming world coordinates is RAS which I believe matches the scanner space DICOM expects, Nifti are generally in RAS as well and is the way [Nibabel produces outputs](https://nipy.org/nibabel/coordinate_systems.html#nibabel-always-uses-an-ras-output-space).

Reply:
Thanks @ericspod